business_requirement_deliverable_resource_template/models/business_requirement_resource_template.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+# © 2017 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import api, fields, models
+from openerp.exceptions import ValidationError
+from openerp.tools.translate import _
+
+
+class BusinessRequirementDeliverable(models.Model):
+    _inherit = "business.requirement.deliverable"
+
+    def _prepare_resource_lines(self):
+        rl_data = self.product_id.sudo().resource_lines.copy_data()
+        return [(0, 0, item) for index, item in enumerate(rl_data)]
```

Comment:
@dreispt I tested returning a `recordset` and the result is that is that the new RL are not saved on the DL so IMO doesn't work the same.

If non an issue could be as is it now?